codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -617,7 +617,7 @@ public ChannelFuture resetStream(final ChannelHandlerContext ctx, int streamId,
         }
 
         final ChannelFuture future;
-        if (stream.state() == IDLE || connection().local().created(stream)) {
```

Comment:
Wait ... is this right? I think we still need something here to make sure that the other endpoint doesn't know about the stream (see https://github.com/netty/netty/pull/4502). Or can we not get into `IDLE` via a remote-initiated stream?